### PR TITLE
Update missing Chinese translation

### PR DESCRIPTION
**Pull request by ly5156**
## Description

Update part of Chinese missing docs for the latest and version 2.8 which includes:

1. manage clusters
2. kubernetes cluster setup
3. infrastructure setup
4. kubernetes clusters in rancher setup

### CODE REVIEW

**Comment by ly5156** [comment]
@JacieChao 
Please help me review this PR. Thank you.

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/kubernetes-clusters-in-rancher-setup.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Rancher中的 Kubernetes集群设置
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/kubernetes-clusters-in-rancher-setup.md:2]: Rancher后，Kubernetes后需要有空格，格式注意下

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/migrate-to-an-out-of-tree-cloud-provider/migrate-to-out-of-tree-vsphere.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+title: 将 vSphere  从树内迁移到树外
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/migrate-to-an-out-of-tree-cloud-provider/migrate-to-out-of-tree-vsphere.md:2]: vSphere 后空格过多，需注意格式

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/manage-clusters/access-clusters/access-clusters.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/how-to-guides/new-user-guides/manage-clusters/access-clusters"/>
+</head>
+
+本节介绍可以用来访问 Rancher 管理的集群的工具。
+
+有关如何授予用户访问集群的权限的信息，请参阅[将用户添加到集群](add-users-to-clusters.md)部分。
+
+有关 RBAC 的更多信息，请参阅[本节](../../authentication-permissions-and-global-configuration/manage-role-based-access-control-rbac/manage-role-based-access-control-rbac.md)。
+
+有关如何设置身份验证系统的信息，请参阅[本节](../../authentication-permissions-and-global-configuration/authentication-config/authentication-config.md)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/manage-clusters/access-clusters/access-clusters.md:15]: 身份验证系统 => 认证

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8.json [hunk 4]
```diff
@@ -1,391 +1,398 @@
 {
-    "version.label": {
-      "message": "v2.8",
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.8.json:3]: 这个文件建议只修改部分内容

**Review by JacieChao — COMMENTED** [review]
Please update the problem with both latest and v2.8 docs

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/kubernetes-clusters-in-rancher-setup.md (hunk 1 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/kubernetes-clusters-in-rancher-setup.md]: 已经修改

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/migrate-to-an-out-of-tree-cloud-provider/migrate-to-out-of-tree-vsphere.md (hunk 2 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/migrate-to-an-out-of-tree-cloud-provider/migrate-to-out-of-tree-vsphere.md]: 已经修改

File: i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/manage-clusters/access-clusters/access-clusters.md (hunk 3 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/current/how-to-guides/new-user-guides/manage-clusters/access-clusters/access-clusters.md]: 已经修改

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8.json (hunk 4 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.8.json]: 已经修改

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/infrastructure-setup/infrastructure-setup.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---
+title: 没有 Kubernetes 集群的基础设施？试试这些教程。
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/how-to-guides/new-user-guides/infrastructure-setup"/>
+</head>
+
+要为带有外部数据库的高可用性 K3s Kubernetes 集群设置基础设施，请参阅本[页面](ha-k3s-kubernetes-cluster.md)。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/infrastructure-setup/infrastructure-setup.md:9]: 这个文件与current下的不一致

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/kubernetes-cluster-setup/kubernetes-cluster-setup.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+title: "没有 Kubernetes 集群？试试这些教程。"
+---
+
+<head>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/zh/how-to-guides/new-user-guides/kubernetes-cluster-setup"/>
+</head>
+
+本节介绍如何安装一个用于运行 Rancher 服务的 Kubernetes 集群。
```
Comment (JacieChao) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/kubernetes-cluster-setup/kubernetes-cluster-setup.md:9]: 这个文件与current下的不一致

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/infrastructure-setup/infrastructure-setup.md (hunk 5 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/infrastructure-setup/infrastructure-setup.md]: 已经同步

File: i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/kubernetes-cluster-setup/kubernetes-cluster-setup.md (hunk 6 above)
Comment (ly5156) [reply i18n/zh/docusaurus-plugin-content-docs/version-2.8/how-to-guides/new-user-guides/kubernetes-cluster-setup/kubernetes-cluster-setup.md]: 已经同步

**Review by JacieChao — COMMENTED** [review]
LGTM.
PR will be ready after other dependencies are ready

**Comment by martyav** [comment]
@ly5156 I was about to merge but there's a conflict. From what I can tell, Git is complaining about the spacing in the edited sections of `i18n/zh/docusaurus-plugin-content-docs/version-2.8.json`. I'll let you have a look and resolve it. Thanks!

**Comment by ly5156** [comment]
> @ly5156 I was about to merge but there's a conflict. From what I can tell, Git is complaining about the spacing in the edited sections of `i18n/zh/docusaurus-plugin-content-docs/version-2.8.json`. I'll let you have a look and resolve it. Thanks!

I rebased the code and resolved the conflicting file

**Pull request merged**